releases/v0.6.0.md

```diff
@@ -0,0 +1,65 @@
+# Release Notes for v0.6.0
+Release Date: June 7th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
```

Comment:
I'd remove the dot at the end of the 2 last bullet points to be consistent with the previous 3 points

Reply:
This bullet is a sentence where the others are not.